src/registrar/tests/common.py

```diff
@@ -577,22 +581,85 @@ def __init__(
         contacts=[],
         hosts=["fake.host.com"],
     )
+    # infoDomainUpdateFail = fakedEppObject(
+    #     "security",
+    #     cr_date=datetime.datetime(2023, 5, 25, 19, 45, 35),
+    #     contacts=[],
+    #     hosts=["ns1.failednameserver.gov"],
+    #     addrs=["1.2.3"],
+    # )
+    infoDomainThreeHosts = fakedEppObject(
+        "my-nameserver.gov",
+        cr_date=datetime.datetime(2023, 5, 25, 19, 45, 35),
+        contacts=[],
+        hosts=[
+            "ns1.my-nameserver-1.com",
+            "ns1.my-nameserver-2.com",
+            "ns1.cats-are-superior3.com",
+        ],
+    )
+    infoDomainNoHost = fakedEppObject(
+        "my-nameserver.gov",
+        cr_date=datetime.datetime(2023, 5, 25, 19, 45, 35),
+        contacts=[],
+        hosts=[],
+    )
+    infoDomainTwoHosts = fakedEppObject(
+        "my-nameserver.gov",
+        cr_date=datetime.datetime(2023, 5, 25, 19, 45, 35),
+        contacts=[],
+        hosts=["ns1.my-nameserver-1.com", "ns1.my-nameserver-2.com"],
+    )
     mockDataInfoContact = fakedEppObject(
         "anotherPw", cr_date=datetime.datetime(2023, 7, 25, 19, 45, 35)
     )
     mockDataInfoHosts = fakedEppObject(
+        "lastPw",
+        cr_date=datetime.datetime(2023, 8, 25, 19, 45, 35),
+        addrs=["1.2.3", "2.3.4"],
+    )
+
+    mockDataHostChange = fakedEppObject(
         "lastPw", cr_date=datetime.datetime(2023, 8, 25, 19, 45, 35)
     )
+    infoDomainHasIP = fakedEppObject(
```

Comment:
Is this the right name for this object? I don't see an IP here at all.

Reply:
Great catch!